osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -1017,6 +1109,14 @@ public virtual bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
                 case GlobalAction.Select:
                     FinaliseSelection();
                     return true;
+
+                case GlobalAction.IncreaseSpeed:
+                    increaseSpeed();
+                    return true;
+
+                case GlobalAction.DecreaseSpeed:
+                    decreaseSpeed();
+                    return true;
```

Comment:
`increaseSpeed()` and `decreaseSpeed()` should probably be one method that accept a delta or something.